plugins/evernote.koplugin/main.lua

```diff
@@ -463,23 +474,29 @@ function EvernoteExporter:exportBooknotesToHTML(title, booknotes)
 end
 
 function EvernoteExporter:exportBooknotesToTXT(title, booknotes)
-    local file = io.open(self.clipping_dir .. "/KOReaderClipping.txt", "a")
+    -- Use wide_space to avoid crengine to treat it specially.
+    local wide_space = "\227\128\128"
+    local file_modification = lfs.attributes(self.text_clipping_file, "modification") or 0
+    local file = io.open(self.text_clipping_file, "a")
     if file then
-        file:write(title .. "\n")
+        file:write(title .. "\n" .. wide_space .. "\n")
         for _ignore1, chapter in ipairs(booknotes) do
             if chapter.title then
-                file:write(" - " .. chapter.title .. "\n\n")
+                file:write(wide_space .. chapter.title .. "\n" .. wide_space .. "\n")
             end
             for _ignore2, clipping in ipairs(chapter) do
-                file:write(T(_("   -- Page: %1, added on %2\n\n"),
-                             clipping.page, os.date("%c", clipping.time)))
-                if clipping.text then
-                    file:write(clipping.text)
-                end
-                if clipping.image then
-                    file:write(_("<An image>"))
+                -- If this clipping has already been exported, we ignore it.
+                if clipping.time >= file_modification then
+                    file:write(T(_(wide_space .. wide_space .. "-- Page: %1, added on %2\n"),
```

Comment:
you will have to move the wide_space concatenation outside of `_()`, otherwise the translation won't work.